w_common/lib/src/common/disposable.dart

```diff
@@ -599,8 +557,6 @@ class Disposable implements _Disposable, DisposableManagerV7, LeakFlagger {
     return completer;
   }
 
-  @mustCallSuper
-  @override
   void manageDisposable(Disposable disposable) {
```

Comment:
`DisposableManager.manageDisposable` was marked as deprecated and it was indicated it would be removed in v2. That makes me think we should remove this implementation too. Would you agree?

Reply:
Oh, I see in the PR description now that we decided to keep this method. Is there a reason why?

Reply:
We discussed this in standup many days ago. If I recall correctly it was:
1. The replacement is to use `manageAndReturnTypedDisposable`, which if you already have an instance of your disposable stored in your local context, isn't exactly what you want. Technically it would work fine, since the return value can be ignored, but the ergonomics are a little funky. 
2. There's literally hundreds of usages of `manageDisposable` at the org, which would slow down adoption of a v2 somewhat. Not too significantly, since we can roll out a simple codemod + sg batch change, but it is worth noting.

Reply:
Sounds reasonable to me! 👍